tests/unit/APNSRequestMetadataTest.php

```diff
@@ -79,4 +79,124 @@ public function testThatUuidSetterWorks() {
 		$meta = $this->new_metadata()->setUuid( $uuid );
 		$this->assertEquals( $uuid, $meta->getUuid() );
 	}
+
+	public function testThatMetadataSerializationIncludesTopic() {
+		$topic = $this->random_string();
+		$encoded = $this->from_json( $this->new_metadata( $topic )->toJSON() );
+		$this->assertEquals( $topic, $encoded->topic );
+	}
+
+	public function testThatMetadataSerializationIncludesUuid() {
```

Comment:
PHP naming question: `Uuid` vs `UUID`. UUID being an acronym (well, sort of) languages like Swift would recommend to write it all in capital letters. Does PHP use only the first one in camel case style?

- `Uuid`
- `Json`
- `Url`
- etc

?

Reply:
I don't think there's a best practice for this – I just find that properCamelCasing leaves the method name looking nice 🤷‍♂️